scripts/package_building/build_artifacts.sh

```diff
@@ -590,8 +591,6 @@ function main {
     GIT_BRANCH="$(get_branch_from_ini "$GIT_BRANCH" "$INI_FILE")"
 
     BASE_DESTINATION_PATH=$DESTINATION_PATH
```

Comment:
is there the need of two variables here?

Reply:
BASE_DESTINATION_PATH is used below to link the last artifacts folder to the latest folder